### DISCUSSION TITLE
Any point in phasing the input data?

### DISCUSSION THREAD

**Discussion by AWS-crafter** [PGScatalog/pgsc_calc#197] [General]
Is there any rationale to phasing the imputed data before it is input into pgsc_calc? I can't think of a reason, but wanted to check.

(I mean for the purposes of pgsc_calc. Phasing may improve imputation quality, which in turn improves PRS accuracy.)

**Comment by smlmbrt** [comment]
Not in terms of the PGS calculation at the moment, but as you said it often improves quality which is ideal for genotyping accuracy. There are methods that calculate PGS on haplotypes (see _partial PGS,_ so phasing would be essential for those), but that is quite rare at the moment.
